### PR TITLE
Sebankid qr

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

File: src/components/SEBankIDSameDeviceButton.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { PKCE, AuthorizeResponse, savePKCEState } from '@criipto/auth-js';
+import { PKCE, AuthorizeResponse } from '@criipto/auth-js';
 import React, {useCallback, useContext, useEffect, useState} from 'react';
-import CriiptoVerifyContext from '../context';
+import CriiptoVerifyContext from '..//context';
```
Comment (sgryt) [inline src/components/SEBankIDSameDeviceButton.tsx:3]: Double '/' intented ?

Comment (mickhansen) [reply src/components/SEBankIDSameDeviceButton.tsx]: No, fixed :)

**Comment by sgryt** [comment]
I'm getting CORS errors in the preview

**Comment by sgryt** [comment]
> I'm getting CORS errors in the preview

Updated app now.

**Comment by mickhansen** [comment]
> > I'm getting CORS errors in the preview
> 
> Updated app now.

Ah, was curious why the callback URLs were already registered when I went to register them :D

**Pull request merged**